### PR TITLE
chore: remove api v1beta2

### PR DESCRIPTION
**Pull request by kiragoo**

### CODE REVIEW

**Comment by Rory-Z** [comment]
Do not delete `.github/workflows/upgrade.yaml`, should change `v1beta2` to `v1beta3` in this file

File: .github/workflows/upgrade.yaml [hunk 1]
```diff
@@ -1,154 +0,0 @@
...
-  upgrade:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        install:
-          - static
-          - helm
-        apiVersion:
-          - v1beta2
```
Comment (Rory-Z) [inline .github/workflows/upgrade.yaml:20]: Do not delete this file, should change v1beta2 to v1beta3 in this line

**Review by Rory-Z — CHANGES_REQUESTED** [review]

File: .github/workflows/upgrade.yaml (hunk 1 above)
Comment (kiragoo) [reply .github/workflows/upgrade.yaml:20]: done

**Review by Rory-Z — APPROVED** [review]
Good job

**Pull request merged**